backend/tests/test_classify_tests.py

```
@@ -0,0 +1,494 @@
+        @pytest.mark.parametrize("x", [1, 2])
+        @pytest.mark.skip(reason="wip")
+        def test_only_builtins(x):
+            assert True
+        """,
+    )
+    scan = classify_tests.scan_file(f)
+    assert scan.tests[0].markers == ()  # neither classifier present
+
+
+@pytest.mark.unit
+def test_classifier_exit_code_clean_repo(tmp_path, capsys):
```

Ruff (0.15.13)

[warning] Unused function argument: `tmp_path`

(ARG001)